.gitignore

```diff
@@ -6,6 +6,7 @@ TestResults/
 _ReSharper.*/
 packages/
 artifacts/
+fakeinstall/
```

Comment:
How about we move this to the `artifacts` folder so that all the output are centralized in one place?